components/components_CSS/Button/button.css

```diff
@@ -0,0 +1,103 @@
+@import url("/root.css");
+
+button{
+    border: none;
+}
+
+.btn{
+    
+    padding: 0.8rem 2.5rem;
+    color: white;
+    border-radius:  8px; ;
+    font-family: Georgia, 'Times New Roman', Times, serif;
```

Comment:
you can define font in your root element instead or here.

Reply:
I used different font inside button thats why i used font here, will try to abstract it